public/index.html

```diff
@@ -5,79 +5,108 @@
+    <link rel="stylesheet" href="./css/font-awesome.min.css">
     <link href="https://fonts.googleapis.com/css?family=Sarina" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
 
     <title>Project Fluent</title>
   </head>
   <body>
-    <div class="main">
-      <header class="header">
-        <h1 class="header__wordmark">
-          fluent
+    <div>
+      <nav id="topbar">
+        <div class="logo">
+          <a href="/">fluent</a>
+        </div>
+        <ul>
+          <li>
+            <a href="#">Home</a>
+          </li>
+          <li>
+            <a href="/fluent/guide">Documentation</a>
+          </li>
+          <li>
+            <a href="/play">Online Editor</a>
+          </li>
+          <li>
+            <a href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+    <div>
+      <header id="intro">
+        <h1>
+          Fluent
         </h1>
-        <nav class="header__nav">
-          <a class="link" href="http://projectfluent.io/fluent/guide">Syntax Guide</a>
-          <a class="link" href="http://projectfluent.io/play">Online Editor</a>
-          <a class="link" href="https://github.com/projectfluent">GitHub</a>
-          <a class="link" href="https://groups.google.com/forum/#!forum/mozilla.tools.l10n">Mailing list</a>
-          <a class="link" href="https://github.com/projectfluent/fluent/wiki">Wiki</a>
-        </nav>
+        <h2>A localization paradigm designed to unleash the expressive power of the natural language.</h2>
+        <a href="https://github.com/projectfluent/fluent/wiki/Integrating-Fluent" class="cta">Get Started</a>
       </header>
-
-      <section class="section section--centered">
-        <p class="pitch">
-
-          <strong class="highlight">Fluent</strong> is a localization paradigm
-          designed to unleash the expressive power of the natural language.
-
+    </div>
+    <div>
+      <main>
+        <ul>
+          <li>
+            <h1>
+              Innovative
+            </h1>
+            <p>Natural-sounding translations that use the grammatical features of each language..</p>
+          </li>
+          <li>
+            <h1>
+              Powerfull & Extensible
+            </h1>
+            <p>
+              Date, time, number, plural rules support. Full bidirectionality.
+              Any internationalization formatters can be added.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Flexible
+            </h1>
+            <p>
+              Can translate on client-side and server-side. Live language switching,
+              robust error handling.
+              JS, Python, Rust and React implementation.
+            </p>
+          </li>
+          <li>
+            <h1>
+              Complete
+            </h1>
+            <p>
+              Visual localization editor, linters, checkers and
+              workflow tooling.
```

Comment:
We don't have any of this right now. Please remove this section.

Reply:
I think that three sections look visually unbalanced. It would be nice to add a fourth if you have an idea for it.
If not, we can land as-is and add the fourth once we have more tooling :)